powerhome/playbook
# React Circle Chart

#### Screens
![Screen Shot 2020-09-17 at 9 21 33 AM](https://user-images.githubusercontent.com/9158723/93475925-30417d00-f8c7-11ea-8beb-196dc67d0c37.png)

#### Breaking Changes
No.

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUXE-143


#### How to test this
1) Dev docs react page.
2) Also, for making sure nothing was affected by a classname change that I had to make to get past Lint Errors (not warning, but error)... please check the following pages in nitro which is the only spot the circle chart is currently being used. There should be nothing broken about them at all, because I verified that we are not using kits incorrectly, and I looked at their scss to make sure they were not targeting any classnames from the kit. 

https://github.com/powerhome/nitro-web/blob/2f46e50cc69948659b9067f766874ed18d6426ef/components/contact_center/app/views/contact_center/agent_stats/_stats.html.erb

https://github.com/powerhome/nitro-web/blob/2f46e50cc69948659b9067f766874ed18d6426ef/components/contact_center/app/views/contact_center/agent_stats/show.html.erb

#### Checklist:

- [X] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
